package.json

```diff
@@ -7,9 +7,9 @@
     "postinstall": "npm run copy-libs",
     "update-deps": "npm update",
     "postupdate-deps": "npm run copy-libs",
-    "copy-libs": "cpx \"node_modules/{angular,angular-*,html5-boilerplate/dist}/**/*\" app/lib -C",
+    "copy-libs": "cpx \"node_modules/{angular,angular-*,@uirouter/angularjs}/**/*\" app/lib -C",
     "prestart": "npm install",
-    "start": "http-server -a localhost -p 8000 -c-1 ./app",
+    "start": "concurrently --kill-others \"sass --watch app/scss/application.scss:app/application.css\" \"pug app -w\" \"http-server -a localhost -p 8000 -c-1 ./app\"",
```

Comment:
mmmm why this kill others?

Reply:
With --kill-others switch, all commands are killed if one dies